gobblin-data-management/src/test/resources/avroToOrcQueryUtilsTest/arrayWithinRecordWithinArrayWithinRecord_nested.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Would be useful to add some comments in this schema to describe what this test does